# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1542656535 28800
#      Mon Nov 19 11:42:15 2018 -0800
# Node ID 9e5caa6acb00fea3f1d3565207af98b531360e39
# Parent  7a8f895c8f152e44a7aa15d7e7379287d09da981
include a DOAP descriptor (bug #53619)

* octave.doap: New file.
* Makefile.am (EXTRA_DIST): Include it in the list.

diff --git a/Makefile.am b/Makefile.am
--- a/Makefile.am
+++ b/Makefile.am
@@ -122,16 +122,17 @@ BUILT_DISTFILES += \
   INSTALL.OCTAVE
 
 EXTRA_DIST += \
   CITATION \
   COPYING \
   INSTALL \
   NEWS \
   README \
+  octave.doap \
   run-octave.in \
   $(BUILT_DISTFILES)
 
 AUTOCONF_SUBST_VARS = @AUTOCONF_SUBST_VARS@
 ALL_SUBST_VARS = \
   $(AUTOCONF_SUBST_VARS) \
   abs_top_builddir \
   abs_top_srcdir \
diff --git a/octave.doap b/octave.doap
new file mode 100644
--- /dev/null
+++ b/octave.doap
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Project xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+         xmlns:foaf="http://xmlns.com/foaf/0.1/"
+         xmlns="http://usefulinc.com/ns/doap#">
+
+  <name xml:lang="en">GNU Octave</name>
+  <shortname>octave</shortname>
+
+  <shortdesc xml:lang="en">Interactive programming environment for numerical computations</shortdesc>
+
+  <homepage rdf:resource="https://www.octave.org" />
+  <license rdf:resource="https://www.gnu.org/licenses/gpl-3.0.txt" />
+  <bug-database rdf:resource="https://savannah.gnu.org/bugs/?group=octave" />
+  <download-page rdf:resource="https://ftp.gnu.org/gnu/octave/" />
+  <mailing-list rdf:resource="https://lists.gnu.org/mailman/listinfo/help-octave" />
+  <wiki rdf:resource="https://wiki.octave.org" />
+
+  <programming-language>C++</programming-language>
+
+  <maintainer>
+    <foaf:Person>
+      <foaf:name>John W. Eaton</foaf:name>
+      <foaf:mbox rdf:resource="mailto:jwe@octave.org" />
+    </foaf:Person>
+  </maintainer>
+
+  <repository>
+    <HgRepository>
+      <browse rdf:resource="https://hg.savannah.gnu.org/hgweb/octave"/>
+      <location rdf:resource="https://hg.savannah.gnu.org/hgweb/octave"/>
+    </HgRepository>
+  </repository>
+
+</Project>
